src/app/containers/RadioSchedule/Canonical/index.jsx

```diff
@@ -120,24 +123,26 @@ const CanonicalRadioSchedule = ({ endpoint }) => {
   const nextLabel = pathOr('NEXT', ['media', 'nextLabel'], translations);
 
   useEffect(() => {
-    const handleResponse = async (response) => {
-      const radioScheduleData = await response.json();
-      const timeOnClient = parseInt(moment.utc().format('x'), 10);
-      const schedules = processRadioSchedule(
-        radioScheduleData,
-        service,
-        timeOnServer || timeOnClient,
-      );
-      setRadioSchedule(schedules);
-    };
-
-    const fetchRadioScheduleData = (pathname) =>
-      fetch(pathname, { mode: 'no-cors' })
-        .then(handleResponse)
-        .catch((e) => logger.error(`HTTP Error: "${e}"`));
-
-    fetchRadioScheduleData(endpoint);
-  }, [endpoint, locale, script, service, timeOnServer, timezone]);
+    if (!schedule) {
+      const handleResponse = async (response) => {
+        const radioScheduleData = await response.json();
+        const timeOnClient = parseInt(moment.utc().format('x'), 10);
+        const schedules = processRadioSchedule(
+          radioScheduleData,
+          service,
+          timeOnServer || timeOnClient,
+        );
+        setRadioSchedule(schedules);
+      };
+
+      const fetchRadioScheduleData = (pathname) =>
+        fetch(pathname, { mode: 'no-cors' })
+          .then(handleResponse)
+          .catch((e) => logger.error(`HTTP Error: "${e}"`));
```

Comment:
Update this to the new error format https://github.com/bbc/simorgh/blob/latest/src/app/lib/README.md#logging